conf/fluo-deploy.props.example

```diff
@@ -13,6 +13,8 @@
 # limitations under the License.
 
 [general]
+# type of cloud service
```

Comment:
I like the change to `ec2` but search in ansible code as there are some references to `aws-ec2` that need to be changed.